src/mobile/components/Orders/index.tsx

```diff
@@ -0,0 +1,89 @@
+import * as React from 'react';
+import { useIntl } from 'react-intl';
+import { useDispatch, useSelector } from 'react-redux';
+import { CloseIcon } from '../../../assets/images/CloseIcon';
+import { TabPanel } from '../../../components';
+import { useUserOrdersHistoryFetch } from '../../../hooks';
+import {
+    ordersCancelAllFetch,
+    ordersHistoryCancelFetch,
+    selectOrdersHistory,
+    selectShouldFetchCancelAll,
+    selectShouldFetchCancelSingle,
+} from '../../../modules';
+import { OrdersItem } from './OrdersItem';
+
+const userOrdersHistoryTabs = ['open', 'all'];
+
+const OrdersComponent = () => {
```

Comment:
Add type for component: (For all)

    const OrdersComponent: React.FC = () ....